TreasureData/TDClient.h

```diff
@@ -7,7 +7,11 @@
 //
 
 #import <Foundation/Foundation.h>
+#if TD_SWIFT_PACKAGE
+@import KeenClientTD;
+#else
 #import "KeenClientTD/KeenClient.h"
```

Comment:
When I tried to create a new project and add packge from nacho4d/td-ios-sdk, I get `'KeenClientTD/KeenClient.h' file not found` error here. Seems like for some reason, TD_SWIFT_PACKAGE is not defined. The ci_spm_build script builds successfully though.

Reply:
@tung-vu-td Thank you for reviewing this. 

I have noticed this issue too. 

I have realized that TD_SWIFT_PACKAGE definition was not correct so fixing it solves half of the problem. With this definition TDUtils.m compiles OK. Since it is am `m` source the flag is applied to it.
<img width="575" alt="スクリーンショット 2022-09-21 0 45 37" src="https://user-images.githubusercontent.com/111651/191304150-00b24b3d-38c7-4938-892f-29c51766b494.png">

However the same definition does not work for `h` sources since they remain as sources even when after added as a target into an app.

I still have no solution other than improving/modifing current sources to use `@import ...` instead of `#import ...` for all builds, even for cocoa pods, etc. I am still thinking ...

Reply:
@tung-vu-td 
I am not an expert in cocoa pods, I wonder if we can use `@import ...` directives even in cocoa pods builds?

Reply:
Hi @nacho4d, Thanks for the reply. Yes, the `@import` solution possible and currently I'm inclined to that solution too. The only downside is that customer has to declare `use_framework!` in their Podfile, which I do think is only a minor nuisance though. Without other solution, I think we will have to go with that.